frontends/election-manager/src/components/print_button.tsx

```diff
@@ -66,7 +72,31 @@ export function PrintButton({
       document.title = documentTitle;
     }
 
+    setShouldRenderPrintTarget(false);
     afterPrint?.();
+  }, [
+    afterPrint,
+    afterPrintError,
+    copies,
+    isMounted,
+    makeCancelable,
+    printer,
+    sides,
+    title,
+  ]);
+
+  useEffect(() => {
+    if (shouldRenderPrintTarget) {
+      void print();
+    }
+  }, [shouldRenderPrintTarget, print]);
+
+  async function startPrint() {
```

Comment:
This should be wrapped in `useCallback`.

Reply:
You mean that `startPrint` should be wrapped in `useCallback`, since it has dependencies on props?

Reply:
No, because it is passed to a JSX prop.